pages/linux/ip-link.md

```diff
@@ -0,0 +1,28 @@
+# ip link
+
+> Manage network interfaces.
+> More information: <https://github.com/dmbaturin/iproute2-cheatsheet>.
```

Comment:
Wouldn't this be better?  It's the deployed site.
```suggestion
> More information: <https://baturin.org/docs/iproute2>.
```

Reply:
I'm actually not a fan of that either. It covers a lot more than `ip link`. How about <https://man.archlinux.org/man/ip-link.8>?

Reply:
Hmm, that man page is actually really full, I guess we should probably use it.

Reply:
What if we used https://baturin.org/docs/iproute2/#ip-link? I think it's a good intermediary step between the tldr page and the man page.

I guess I still don't understand when a link to a non-man page is appropriate.

Reply:
Well @ahert you're not the only one learning 😄 

In this case the man page linked above is quite full of info and it even has an [examples section](https://man.archlinux.org/man/ip-link.8#EXAMPLES) at the bottom, so I think it definitely qualifies as a good "more information" page.

Reply:
OK. I'll change it to point to a man page.

I'm about to submit a pull request for ip-route. I don't think the ip-route man page has a particularly good Examples section. Would linking to baturin.org be appropriate in that case?

Reply:
Well, that man page does have an examples section, and there's also lots of explanation about the various options and values, so I'd say use the man page for that one as well.

Not all of the manpages are that good, but these ones seem to be prett informative.